content/docs/learning-guide/samples/advanced/_index.md

```diff
@@ -10,3 +10,4 @@ Advanced projects are made and maintained over a long period of times. They are
 | Image | Link | Description | Last Updated |
 | - | - | - | - |
 | <img src="/images/learning-guide/samples/advanced/multiplayer-sample.png" width="300px" /> | [Multiplayer Sample](https://github.com/o3de/o3de-multiplayersample) | Third-person multiplayer game where robots battle one another for dominance in an under construction, multi-tiered starbase | O3DE **23.10.3**. April 28, 2024 |
+| <img src="/images/learning-guide/samples/advanced/starter-game.jpg" width="300px" /> | [Starter Game](https://github.com/o3de/startergame-assets) | This project is a port to O3DE of the [Lumberyard "Starter Game"](https://aws.amazon.com/blogs/gametech/now-available-starter-game/) project. It is still a work-in-progress, but once over it will feature a large map with vegetation, third person controller and enemy AI | O3DE **24.09.0**. September 01, 2024 |
```

Comment:
Just a minor nit:

"over" might be better as "finished" or "complete", and should have a comma following it.